syntax = "proto3";

package pb;

import "pb/rpc_signin_user.proto";
import "pb/rpc_signup_user.proto";
import "pb/user.proto";

option go_package = "cyrex/vacancies/pb";

service AuthService {
  rpc SignUpUser(SignUpUserInput) returns (GenericResponse) {}
  rpc SignInUser(SignInUserInput) returns (SignInUserResponse) {}
  rpc VerifyEmail(VerifyEmailRequest) returns (GenericResponse) {}
}

message VerifyEmailRequest { string verificationCode = 1; }